modules/navier_stokes/test/tests/finite_volume/pins/channel-flow/2d-rc-friction.i

```diff
@@ -159,12 +159,6 @@ velocity_interp_method='rc'
     variable = v
     function = 0
   []
-  [inlet-p]
-    type = INSFVOutletPressureBC
-    boundary = 'left'
-    variable = pressure
-    function = 1
-  []
```

Comment:
What was going on here before and why didn't this cause any exodiffs?

Reply:
It wasnt used in this test. 
I was experimenting with using INSFVOutletPressureBC at an inlet to drive the flow through a dP. It works for fully developed if I remember correctly.

Reply:
This came about because someone was re-using this input file without selecting what boundary conditions were active. And that gave very bad solutions.